.github/workflows/e2e.yaml

```diff
@@ -47,7 +47,33 @@ jobs:
       - name: Harden Runner
         uses: step-security/harden-runner@5c7944e73c4c2a096b17a9cb74d65b6c2bbafbde # v2.9.1
         with:
-          egress-policy: audit
+          disable-telemetry: true
+          egress-policy: block
+          allowed-endpoints: >
+            *.azmk8s.io:443
+            *.azurecr.io:443
+            *.data.mcr.microsoft.com:443
+            aka.ms:443
+            api.github.com:443
+            app.aladdin.microsoft.com:443
+            auth.docker.io:443
+            azcliextensionsync.blob.core.windows.net:443
+            clients3.google.com:80
```

Comment:
Skaffold (confirmed), likely [telemetry](https://skaffold.dev/docs/resources/telemetry/), likely can be avoided/blocked with/after `skaffold config set --global collect-metrics false`

Reply:
Can we leave a comment within the actual code for any odd cases?

Reply:
Yes (will add), but not inline due https://github.com/step-security/harden-runner/issues/199